data/card/csvstore/Store.go

```diff
@@ -0,0 +1,67 @@
+package csvstore
+
+import (
+  "github.com/pkg/errors"
+
+  dataCard "github.com/gbrayhan/academy-go-q12021/data/card"
+  domainCard "github.com/gbrayhan/academy-go-q12021/domain/card"
+  domainErrors "github.com/gbrayhan/academy-go-q12021/domain/errors"
+)
+
+const (
+  createError = "Error in creating new card"
+  readError   = "Error in finding card in the database"
+  listError   = "Error in getting cards from the database"
+)
+
+// Store struct manages interactions with cards store
+type Store struct {
+  csv       FileCSV
+  cardsRepo domainCard.CardRepository
+}
+
+func New() *Store {
+  return &Store{
+  }
+}
+
+func (s *Store) CreateCard(cardDom *domainCard.Card) (card *domainCard.Card, err error) {
+
+  return
+}
+
+func (s *Store) ReadCard(id int) (card *domainCard.Card, err error) {
+
+  dCard, err := s.csv.FindCardByID(id)
+
+  if err != nil {
+    err = domainErrors.NewAppError(errors.Wrap(err, readError), domainErrors.RepositoryError)
```

Comment:
I didn't know about `errors.Wrap`. Is this similar to doing `fmt.Errorf("%w", err)`?

Reply:
yes is similar